charts/argo-workflows/values.yaml

```diff
@@ -171,9 +171,11 @@ controller:
     - default
 
   # -- Specifies the container runtime interface to use (one of: `docker`, `kubelet`, `k8sapi`, `pns`, `emissary`)
+  # NOTE: Emissary executor is now the only supported executor from v3.4.0.
```

Comment:
I added some notes for in case. 🙋 
ref: https://argoproj.github.io/argo-workflows/upgrading/#upgrading-to-v34
<img width="733" alt="Screen Shot 2022-09-19 at 21 04 31" src="https://user-images.githubusercontent.com/30188755/191013208-fa7c812f-c53c-488a-aff6-b113e6696cc0.png">